app/src/main/java/de/westnordost/streetcomplete/quests/accepts_cash/AddAcceptsCash.kt

```diff
@@ -13,9 +13,12 @@ class AddAcceptsCash(o: OverpassMapDataAndGeometryApi) : SimpleOverpassQuestType
         "restaurant", "cinema", "nightclub", "planetarium", "theatre", "marketplace",
         "internet_cafe", "car_wash", "fuel", "pharmacy", "telephone", "vending_machine"
     )
-    private val tourism = listOf(
-        "zoo", "aquarium", "theme_park", "museum", "hotel", "hostel", "motel", "guest_house",
-        "apartment", "camp_site", "attraction", "gallery"
+    private val tourismWithoutImpliedFees = listOf(
+        "zoo", "aquarium", "theme_park", "hotel", "hostel", "motel", "guest_house",
+        "apartment", "camp_site"
+    )
+    private val tourismWithImpliedFees = listOf(
```

Comment:
`tourismWithImpliedFees` means (at least to me) "tourism object that imply fee" while the opposite is happennng here

note also https://github.com/westnordost/StreetComplete/pull/2019#discussion_r471097387 - @westnordost proposed `tourismWithoutImpliedFees` name (With vs Without)

Reply:
You know, sometimes I wonder what my brain recognized and relays to the fingers for programming...
Resolved with 555f549

Reply:
> You know, sometimes I wonder what my brain recognized and relays to the fingers for programming...

[Seriously…](https://github.com/lutris/website/pull/489#pullrequestreview-431024760)